src/Text/Pandoc/Writers/CommonMark.hs

```diff
@@ -237,6 +237,10 @@ inlineToNodes opts SoftBreak
   | writerWrapText opts == WrapNone     = (node (TEXT " ") [] :)
   | otherwise                           = (node SOFTBREAK [] :)
 inlineToNodes opts (Emph xs) = (node EMPH (inlinesToNodes opts xs) :)
+inlineToNodes opts (Underline xs)
+  | isEnabled Ext_raw_html opts = ((node (HTML_INLINE (T.pack "<u>")) [] : inlinesToNodes opts xs ++
```

Comment:
Looks like this line is > 80 characters long.

Reply:
I wonder if it would make more sense to fall back to regular Emph behavior when no more specific rendering of underlined text is available -- both here and in other writers?

Reply:
Done.